### DISCUSSION TITLE
How to not surround blockquotes with quotes

### DISCUSSION THREAD

**Discussion by seabiscuit-iv** [satnaing/astro-paper#364] [General]
Does anyone know if its possible to remove the surrounding quotes of blockquotes? I can't seem to find a css property for it or anything.

**Comment by xyzqm** [comment]
This should help: https://github.com/tailwindlabs/tailwindcss-typography/issues/66

The css stems from Tailwind typography, not the AstroPaper theme itself. Hope this helps!
